topic-operator/src/main/java/io/strimzi/operator/topic/Config.java

```diff
@@ -119,6 +120,9 @@ private Value(String key, Type<? extends T> type, boolean required) {
     /** The kubernetes namespace in which to operate. */
     public static final Value<String> NAMESPACE = new Value<>(TC_NAMESPACE, STRING, true);
 
+    /** Cluster-ID */
+    public static final Value<String> CLIENT_ID = new Value<>(TC_CLIENT_ID, STRING, "");
```

Comment:
I think the default should be something more like `"strimzi-topic-operator-" + uuid`, that way it's at least vaguely useful even when not configured.

Reply:
Ok sure@tombentley , I will make this change. Regarding the UUID, can we use UUID.randomUUID() to generate it ?

Reply:
`UUID.randomUUID()` is fine with me.

Reply:
Okay. Thanks

Reply:
Hello @tombentley,  I have made the necessary changes as per mentioned above but I have not changed the tests. Do I need to change them or are they fine ?